tensorflow/lite/arena_planner.cc

```diff
@@ -153,7 +153,7 @@ TfLiteStatus ArenaPlanner::PlanAllocations() {
     }
   }
   // Go through the graph in execution order.
-  for (int i = 0; i < graph_info_->num_nodes(); ++i) {
+  for (size_t i = 0; i < graph_info_->num_nodes(); ++i) {
     const TfLiteNode& node = graph_info_->node(i);
 
     // First queue output tensors for allocation.
```

Comment:
I don't think deallocate(i) takes a size_t?

Reply:
@jdduke , thanks for pointing this out, i updated the code as per the review comments, kindly check and approve.

Regards
Amit